File: java/0.neuron/Main.java
/*
 * 作者：解琛；
 * 功能：基本神经元；
 * 时间：2021 年 3 月 19 日；
 */

public class Main {
    public static void main(String[] args) {
        double weight[] = {1, 0};
        double bias = 4;
    
        double total = 0;
        double inputs[] = {2, 3};

        Neuron neuron = new Neuron(weight, bias);
        total = neuron.feedforward(inputs);

        System.out.printf(">> 神经元的输出结果为：%f\n", total);
    }
}

File: go/0.neuron/main.go
/*
 * 作者：解琛；
 * 功能：基本神经元；
 * 时间：2021 年 3 月 19 日；
 */

package main

import (
	. "fmt"
	. "./neuron"
)

func main() {
	var neuron = NewNeuron([2]float64{1, 0}, 4)
	var output = neuron.Feedforward([2]float64{2, 3})

	Println(">> 神经元的输出结果为：", output)
}
